### ISSUE TITLE
Install php72 with Error: no implicit conversion of String into Array

### ISSUE THREAD

**Issue by ideepblue** [Homebrew/homebrew-php#4766]
## Description

Install php72 with Error. I have successfully installed php72 in the early today.
However, when I try to config httpd (the brew one, not OSX) with php-fpm, the httpd cannot load `load lib/httpd/modules/mod_proxy_fcgi.somodule`. So I try to solve this issue by reinstalling php72 and httpd. Rather than solving the httpd issue, I got this error. 

The result is that php72 has been installed, but I cannot see the info telling me everything is ok. 

I also tried `brew info php72`. I got the same error. Please tell me if php72 has been successful installed? Or if something else went wrong?

## Context and Logs

```bash
$ brew install php72                                                                                                                    23:26:27
==> Installing php72 from homebrew/php
==> Downloading https://homebrew.bintray.com/bottles-php/php72-7.2.2_13.high_sierra.bottle.tar.gz
Already downloaded: /Users/ideepblue/Library/Caches/Homebrew/php72-7.2.2_13.high_sierra.bottle.tar.gz
==> Pouring php72-7.2.2_13.high_sierra.bottle.tar.gz
Error: no implicit conversion of String into Array
Please report this bug:
  https://docs.brew.sh/Troubleshooting.html
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-php/Abstract/abstract-php.rb:614:in `caveats'
/usr/local/Homebrew/Library/Homebrew/caveats.rb:18:in `caveats'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:573:in `caveats'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:599:in `finish'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:343:in `install_formula'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:232:in `block in install'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:230:in `each'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:230:in `install'
/usr/local/Homebrew/Library/Homebrew/brew.rb:100:in `<main>'

```

Parameter | Value
--------- | ------------------
**macOS version:**       | 10.13.3
**Homebrew version:**   | Homebrew 1.5.4 Homebrew/homebrew-core (git revision db996; last commit 2018-02-12)
**PHP version in use:** | PHP 7.2.2 (cli) (built: Feb  1 2018 11:50:40) ( NTS )
**Xcode version:**      | Xcode 9.2 Build version 9C40b

**Output of `gcc -v`:**

```bash
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 9.0.0 (clang-900.0.39.2)
Target: x86_64-apple-darwin17.4.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

**Output of `php -v`:**
PHP 7.2.2 (cli) (built: Feb  1 2018 11:50:40) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies

**Build logs:**
https://gist.github.com/b0206e4bc5a89839d1689a8ead0ec75b

**Comment by shcode** [comment]
https://github.com/Homebrew/homebrew-php/blob/7d187ab980c3a3628db3bfea748868e8f44d0210/Abstract/abstract-php.rb#L609

This line causes a break when installing. For quick fix, just change `+=` to `<<`

**Comment by ideepblue** [comment]
@shcode Thank you. That works.

**Comment by rammium** [comment]
@shcode Thanks!

**Comment by Kadaaran** [comment]
Saved my day thanks ! 🙏

**Comment by javian** [comment]
This has been fixed now #4769
